### PR TITLE
chore: Prepare Release 1.0.0

### PR DESCRIPTION
**Pull request by miquelbeltran**
### Description :memo:
- **Purpose**: Prepare for release 1.0.0
- **Approach**: Bump version and update changelog

**Type of change**

- [x] Chore

**Updates**

- Run `npm version major --no-git-tag-version`
- Update changelog

### Test plan :test_tube:

- None

### Author to check :eyeglasses:
- [ ] Project and all contained modules builds successfully
- [ ] Self-/dev-tested 
- [ ] Unit/UI/Automation/Integration tests provided where applicable
- [ ] Code is written to standards
- [ ] Appropriate documentation written (code comments, internal docs)

### Reviewer to check :heavy_check_mark:
- [ ] Project and all contained modules builds successfully
- [ ] Change has been dev-/reviewer-tested, where possible
- [x] Unit/UI/Automation/Integration tests provided where applicable
- [x] Code is written to standards
- [ ] Appropriate documentation written (code comments, internal docs)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,29 @@
...
+- feat: #219 Better Send parameters (#221) (2024-05-17)
+- feat: Breadcrumbs (#210) (2024-05-15)
+- chore(deps-dev): bump @stylistic/eslint-plugin from 2.0.0 to 2.1.0 (#214) (2024-05-13)
+- chore(deps-dev): bump tap from 18.7.2 to 18.8.0 (#213) (2024-05-13)
+- chore(deps-dev): bump @types/node from 20.12.8 to 20.12.11 (#212) (2024-05-13)
+- chore(deps-dev): bump semver from 7.6.0 to 7.6.2 (#211) (2024-05-13)
+- docs: Update status badge in README.md (#209) (2024-05-10)
+- refactor: #205 Setup eslint style (#207) (2024-05-09)
+- refactor: #197 Refactor to use Promises internally (#200) (2024-05-09)
+- refactor: #184 Cleanup debug/log messages and styles (#194) (2024-05-08)
```
Comment (sumitramanga) [inline CHANGELOG.md:18]: I think we should order these in a certain way from marketing perspective. I think the order should be - 

1. feat
2. fix
3. refactor
4. chore

The exciting stuff at the top and the not so exciting stuff at the bottom. What do you think?
Not sure if we want to apply this to the other notes as well

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,29 @@
+## 1.0.0
```
Comment (sumitramanga) [inline CHANGELOG.md:1]: This should be pre-release. Do you reckon?

File: package.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "raygun",
   "description": "Raygun package for Node.js, written in TypeScript",
-  "version": "0.15.0-0",
+  "version": "1.0.0",
```
Comment (sumitramanga) [inline package.json:4]: Woohoo 🥳

**Review by sumitramanga — CHANGES_REQUESTED** [review]
A couple of comments to consider/set a standard for :)

File: CHANGELOG.md (hunk 2 above)
Comment (miquelbeltran) [reply CHANGELOG.md:1]: I will ask for clarification on Slack

File: CHANGELOG.md (hunk 1 above)
Comment (miquelbeltran) [reply CHANGELOG.md:18]: I don't mind manually sorting them.

To generate the list, I used `git log --pretty=format:"- %s (%as)"`

Changelogs won't even appear on npmjs.com, so this is only relevant to the user who visits GitHub to find more.

I wouldn't change the already existing changelogs, but only do that from 1.0.0 moving forward.

Comment (miquelbeltran) [reply CHANGELOG.md]: Sorting has been fixed!

**Pull request merged**